docs/_blog/2019-08-21-introduction-to-eclipse-codewind-build-high-quality-cloud-native-applications-faster.md

```diff
@@ -5,7 +5,7 @@ categories: blog
 author_picture: images/blog/author_icon_andy_watson.jpeg
 author_url: 
 author: Andy Watson
-blog_description: "Eclipse Codewind is an open source project that makes it easier for developers to create cloud-native applications within their favorite IDE. Codewind initially supports Visual Studio Code, Eclipse..."
```

Comment:
do you also need to change where these are used in layouts? i.e https://github.com/alif-munim/codewind-docs/blob/alif-search/docs/_layouts/guides.html#L25... I'd do a search on `description` and make sure each is changed.

Reply:
Good catch! I've update the blog layout. Did a search for description as suggested, but wasn't able to find any more.